crates/ra_assists/src/assist_ctx.rs

```diff
@@ -84,6 +84,12 @@ impl<'a, DB: HirDatabase> AssistCtx<'a, DB> {
         f: impl FnOnce(&mut AssistBuilder),
     ) -> Option<Assist> {
         let label = AssistLabel { label: label.into(), id };
+        assert_eq!(
+            label.label.chars().nth(0).and_then(|c| Some(c.is_uppercase())),
```

Comment:
Might as well `unwrap` here, an empty label should be treated as an error I think?

Reply:
r=me otherwise!